### DISCUSSION TITLE
Sell to cover

### DISCUSSION THREAD

**Discussion by mwright-pivotal** [hickeng/financial#154] [General]
first off .. this repo has helped me understand things soo much better!  thank you.   I am curious if anyone else has "Sell to cover" transactions to deal with as well.  Taxes are withheld from those so I am curious if they would still go in the sheet or just report them normally as reported from eTrade?

**Comment by hickeng** [comment]
Vesting RSUs have a `0.00` _cost basis_ (because you paid nothing for them) and a _tax basis_ of the market value at the time they vested.

The reason for the tax basis being non-zero is because RSUs are considered normal income when you receive them, are taxed as such, and show up on your W2. Because you pay tax on their market value on receipt, that's their tax basis.

The way in which tax is paid on the RSUs is:
1. sell-to-cover/withhold-to-cover - the value of the shares used for this is added to your tax paid for the year
2. refund/deficit during filing 1040 - if you over/under paid taxes per your W2, including the value of the withheld shares, it's balanced at filing time.

Generally withheld shares aren't a transaction you need to care about - they never came into your possession in the same way that the federal and state tax withheld from salary never enters your possession and the withheld shares should not show up on your 1099.

If you're talking about something other than sell-to-cover as part of RSU vest, then:
a. I'm not sure in what context that occurs
b. the sheet _only_ handles the impact of the merger on shares you held over the merger date
